sewing-kit.config.ts

```diff
@@ -19,9 +19,6 @@ export default function sewingKitConfig(
       plugins.jest((config: InitialOptions) => {
         config.roots = [join(__dirname, 'src'), join(__dirname, 'tests')];
 
-        config.setupFiles.push(join(tests, 'setup.ts'));
```

Comment:
Sewing-kit now automatically adds these files to the setupFiles and setupFilesAfterEnv blocks if they exist, so we don't need to be explict here.